src/Orleans/Core/ClientBuilder.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.CodeGeneration;
+using Orleans.Messaging;
+using Orleans.Providers;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.Serialization;
+using Orleans.Streams;
+
+namespace Orleans
```

Comment:
Maybe there's a better namespace than the root, but I configuration seems a bit wrong.
ASP.NET's `WebHostBuilder` is in `Microsoft.AspNetCore.Hosting`, so I could put it in a similar namespace.